File: Python/Two Sum II - Input array is sorted/Two Sum II - Input array is sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        lo, hi = 0, len(numbers)-1
        while(numbers[lo]+numbers[hi] != target):
            if(numbers[lo]+numbers[hi] < target):
                lo += 1
            else:
                hi -= 1
        return [lo+1, hi+1]

File: C#/Two Sum II - Input array is sorted/Two Sum II - Input array is sorted.cs
public class Solution {
    public int[] TwoSum (int[] numbers, int target) {
        int lo = 0, hi = numbers.Length - 1;
        while (numbers[lo] + numbers[hi] != target) {
            if (numbers[lo] + numbers[hi] < target) lo++;
            else hi--;
        }
        return new int[2] { lo + 1, hi + 1 };
    }
}